src/complex/Utilities/ParallelAlgorithmUtilities.hpp

```diff
@@ -109,7 +109,5 @@ auto ExecuteParallelFunction(DataType dataType, ParallelRunnerT&& runner, ArgsT&
       return runner.template execute<>(ClassT<float64>(std::forward<ArgsT>(args)...));
     }
   }
-
-  throw std::runtime_error("complex::ExecuteParallelFunction has all supported DataTypes disabled. Error: Invalid DataType");
```

Comment:
No I don't think so, probably just mistakenly got committed after we were playing around with how to template out the stuff I needed.

Reply:
@JDuffeyBQ I would argue so. it's a runtime check for something that is determined at compile time this would be more of an annoyance for users than anything a static_assert would be preferred, but that would be an incredibly long line i think.